examples.md

```diff
@@ -3,21 +3,98 @@ layout: page
 title: All Examples
 permalink: /examples/
 ---
+
+
+<input id="searchInput" class="form-control form-control-lg mb-4" type="text" placeholder="Search" aria-label="Search" aria-describedby="basic-addon1">
+
+
 <div class="examples row">
   {% for example in site.examples reversed %}
-  <div class="col-6 col-lg-4 col-md-6 col-lg-3 pb-5">
+  <div data-id="{{example.id}}" class="example col-6 col-lg-4 col-md-6 col-lg-3 pb-5">
     <h4><a href="{{ example.url }}">{{ example.title }}</a></h4>
 
     {% if example.preview-image %}
-    <div class="bg-secondary p-3 mb-3">
+    <div class="bg-primary p-3 mb-3">
       <a href="{{ example.url }}">
         <img class="mw-100" src="{{ site.baseurl }}/assets/images/examples/{{ example.preview-image }}" alt="{{ example.title }}" />
       </a>
     </div>
     {% endif %}
 
     <p>{{ example.synopsis }}</p>
-    <p><a href="{{ example.url }}">Read full example</a></p>
+    <p><a href="{{ example.url }}">View full example</a></p>
   </div>
   {% endfor %}
+  
+  <!-- Not related in any way to Fuse Opens FuseJS btw -->
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.4.4/fuse.min.js"></script>
+  <script type="text/javascript">
+  $(function() {
+
+    //build the data list to search
+    var list = [
+    {% for example in site.examples reversed %}
+    {
+      id: "{{example.id}}",
+      title: "{{example.title}}",
+      date: "{{example.date}}",
+      tags: "{% for tag in example.tags %}{{ tag }},{% endfor %}",
+      uxConcepts: "{% for concept in example.uxConcepts %}{{ concept }},{% endfor %}",
+      jsConcepts: "{% for concept in example.jsConcepts %}{{ concept }},{% endfor %}"
+    },
+    {% endfor %}
+    ];
```

Comment:
`var list = {{ site.examples | jsonify }}`, I think...

Reply:
Doesn't seem to generate if I use that, maybe because it then compiles the whole page content into json as well, which I don't need, just the searchable content.